src/IO/LzmaWriteBuffer.cpp

```diff
@@ -60,24 +60,31 @@ void LzmaWriteBuffer::nextImpl()
 
     lstr.next_in = reinterpret_cast<unsigned char *>(working_buffer.begin());
     lstr.avail_in = offset();
-    std::cout << lstr.avail_in << std::endl;
+    //std::cout << lstr.avail_in << std::endl;
 
     lzma_action action = LZMA_RUN;
     do {
         out->nextIfAtEnd();
         lstr.next_out = reinterpret_cast<unsigned char *>(out->position());
         lstr.avail_out = out->buffer().end() - out->position();
+        //std::cout << lstr.avail_out << " BEFOR" << std::endl;
+
 
         lzma_ret ret = lzma_code(&lstr, action);
         out->position() = out->buffer().end() - lstr.avail_out;
 
-        if (ret == LZMA_STREAM_END)
-				return;
+        //std::cout << lstr.avail_out << " AFTER" << std::endl;
+
+		//std::cout << ret << " RET IMPL" << std::endl;
+
+        if (ret == LZMA_STREAM_END) {
```

Comment:
Coding style: curly bracket is always on a new line (and not only here but in a few other places) (except for `} while (true)`)